Source/OxyPlot.ImageSharp/ImageRenderContext.cs

```diff
@@ -254,9 +258,10 @@ public override void DrawText(ScreenPoint p, string text, OxyColor fill, string
         }
 
         /// <inheritdoc/>
-        public override OxySize MeasureText(string text, string fontFamily = null, double fontSize = 10, double fontWeight = 500)
+        public override OxySize MeasureText(string text, string? fontFamily = null, double fontSize = 10, double fontWeight = 500)
         {
-            return this.MeasureTextLoose(text, fontFamily, fontSize, fontWeight);
+            // TODO: should fontFamily be nullable?
```

Comment:
Were you able to come to a conclusion on this?

Reply:
The `RenderErrorMessage` logic leaves is `null`, which I presume is intentional to allow the render context to pick a simple font if there are rendering errors, or something like that. The example library is also full of `null` font families, so I don't think we can change it in a hurry. I did spot a bug, though, which I've lazily pushed as a small addition to this PR.